hack/windows/Clean-AntreaNetwork.ps1

```diff
@@ -0,0 +1,47 @@
+<#
+.SYNOPSIS
+Clean OVS bridge and HnsNetwork created by Antrea Agent
+#>
+
+function GetHnsnetworkId($NetName) {
+    $NetList= $(Get-HnsNetwork -ErrorAction SilentlyContinue)
+    if ($NetList -eq $null) {
+        return $null
+    }
+    foreach ($Net in $NetList) {
+        if ($Net.Name -eq $NetName) {
+            return $Net.Id
+        }
+    }
+    return $null
+}
+$AntreaHnsNetworkName = "antrea-hnsnetwork"
+
+Write-Host "Delete OVS bridge: br-int"
+ovs-vsctl.exe --no-wait del-br br-int
+$MaxRetryCount = 10
+$RetryCountRange = 1..$MaxRetryCount
+$BrIntDeleted = $false
+foreach ($RetryCount in $RetryCountRange) {
+    Write-Host "Waiting for OVS bridge deletion complete ($RetryCount/$MaxRetryCount)..."
+    $BrIntAdapter = $(Get-NetAdapter br-int -ErrorAction SilentlyContinue)
+    if ($BrIntAdapter -eq $null) {
+        $BrIntDeleted = $true
+        break
+    }
+    if ($RetryCount -eq $MaxRetryCount) {
+        break
+    }
+    Start-Sleep -Seconds 5
+}
+if (!$BrIntDeleted) {
+    Write-Host "Failed to delete OVS Bridge, please retry the script or do the operations manually."
```

Comment:
"...Bridge. Please..."

Change "or do the operations manually" to "or delete the bridge manually"?

Reply:
Done